### ISSUE TITLE
train data

### ISSUE THREAD

**Issue by hhhhzs666** [stanford-futuredata/ColBERT#152]
My training data looks like this tsv file

![image](https://user-images.githubusercontent.com/110543383/212473464-f6262a40-e3a4-42ed-b534-2e4158e187e1.png)

but Prompt error：

![image](https://user-images.githubusercontent.com/110543383/212473612-a40326d2-38d1-450d-a9c1-51a81591e4df.png)

Could you tell me how to solve this problem, thank you！

**Comment by okhat** [comment]
are you trying to fine-tune an existing checkpoint?We may need more code to understand your problem.

Also, please shuffle your triples before training. They seem very systematically ordered, which is not good for training
